builds/fabric8-che/plugins/keycloak-plugin-token-provider/src/main/java/com/redhat/che/keycloak/token/provider/contoller/TokenController.java

```diff
@@ -105,10 +109,20 @@ public Response getOpenShiftToken(@HeaderParam(HttpHeaders.AUTHORIZATION) String
         try {
             validator.validate(keycloakToken);
             token = tokenProvider.obtainOsoToken(keycloakToken);
-        } catch (KeycloakException e) {
+        } catch (Exception e) {
             return Response.status(Response.Status.BAD_REQUEST).entity(e.getMessage()).build();
         }
         return Response.ok(token).build();
     }
 
+    @GET
+    @Path("/user")
+    public Response getUserMatches(@HeaderParam(HttpHeaders.AUTHORIZATION) String keycloakToken) {
```

Comment:
please add javadoc for this method:

> Add endpoint /api/token/user, which returns true if the token provided in the authorization header corresponds to the user who owns the namespace.

If you like we can create another issue for other PRs
This is in the PR description